cibuild.sh

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+WD=$PWD
+nuttx=$WD/../nuttx
+apps=$WD/../apps
+tools=$WD/../tools
+prebuilt=$WD/../prebuilt
+
+# genromfs
```

Comment:
Maybe `genromfs` can be skipped if already in the distribution. I guess that is the case in the builds.apache.org Jenkins slaves marked as 'ubuntu' (could be just `apt install genromfs` ... it seems is the same https://launchpad.net/ubuntu/+source/genromfs/0.5.2-2build2 , same sha256 hash). Anyway maybe is safer by now to use it from _nuttx/tools_, maybe there is a reason for that I am not aware of.

Reply:
I think there should be sudo permissions to apt install genromfs and others in apache jenkins slaves, it would be hard to get and maintain. So use our owned prebuilt tools may be safer.

Reply:
ok, it makes sense.